### ISSUE TITLE
DPI flag has no value by default, causing failure to execute in lutris 0.5.10.1

### ISSUE THREAD

**Issue by GeorgelT** [lutris/lutris#4504]
``Hello,

lutris recently got upgraded on my system from 0.5.9.1-1 to 0.5.10.1 . This seems to have introduced an issue related to the DPI feature in the GUI and is causing issues even with the newly added uplay connect addon. It manages to connect to the account, but when you try in install anything it fails the same with this issue. 

Error message that is displayed to user: 
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'

I already had DPI settings on most of my games so I didn't notice the issue right away, but any games that don't have it on, or the checkbox is off, or you try to run any of the wine consoles/registry/control panel it will silently fail without any error message.

I started lutris with lutris -d to debug this issue and that's how I managed to find that it was crashing because the other buttons don't have the DPI setting. A listing from the debug area when I click on regedit or wine console.:

```
EBUG    2022-09-13 15:35:33,753 [wine.use_lutris_runtime:115]:/home/andrei/.local/share/lutris/runners/wine/lutris-GE-Proton7-28-x86_64/bin/wine is provided by Lutris, using runtime
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lutris/gui/widgets/game_bar.py", line 264, in on_link_button_clicked
    callback(button)
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 687, in run_regedit
    self._run_executable("regedit")
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 650, in _run_executable
    wineexec(
  File "/usr/lib/python3/dist-packages/lutris/runners/commands/wine.py", line 302, in wineexec
    wine.prelaunch()
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 799, in prelaunch
    self.set_regedit_keys()
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 751, in set_regedit_keys
    prefix_manager.set_dpi(self.get_dpi())
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 761, in get_dpi
    explicit_dpi = int(explicit_dpi)
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'
DEBUG    2022-09-13 15:41:12,693 [wine.use_lutris_runtime:115]:/home/andrei/.local/share/lutris/runners/wine/lutris-GE-Proton7-28-x86_64/bin/wine is provided by Lutris, using runtime
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/lutris/gui/widgets/game_bar.py", line 264, in on_link_button_clicked
    callback(button)
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 671, in run_wineconsole
    self._run_executable("wineconsole")
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 650, in _run_executable
    wineexec(
  File "/usr/lib/python3/dist-packages/lutris/runners/commands/wine.py", line 302, in wineexec
    wine.prelaunch()
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 799, in prelaunch
    self.set_regedit_keys()
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 751, in set_regedit_keys
    prefix_manager.set_dpi(self.get_dpi())
  File "/usr/lib/python3/dist-packages/lutris/runners/wine.py", line 761, in get_dpi
    explicit_dpi = int(explicit_dpi)
TypeError: int() argument must be a string, a bytes-like object or a real number, not 'NoneType'

```

It does not matter which runner I use because it's erroring out before it even gets to the wine executable from the GUI.

**Comment by GeorgelT** [comment]
Would also like to add that doing the downgrade to 5.9.1 fixes this issue. And for completeness I'm on Mint 21/Ubuntu 22.04. I have lutris installed through the official ppa like described on the main page.

**Comment by danieljohnson2** [comment]
That's a bug indeed, but it was fixed in version 0.5.11 released recently (PR #4254, Thanks @SwimmingTiger)

0.5.9.1 didn't have the DPI setting at all, so that's why it can't crash.

Many distributions are already shipping this, but Ubuntu is not, and it appears the Lutris PPA doesn't have it either. You could download the .deb off [GitHub here](https://github.com/lutris/lutris/releases), if you can handle installing that, or clone the master branch and run it directly (via the bin/lutris script within it)
